### PR TITLE
fix(comment):- Fix comment reply

### PR DESCRIPTION
**Pull request by kamal09rumba**
- Disable reply on more then two levels
- Fix double click unselect on survey sentiment and survey review component

### CODE REVIEW

File: src/components/SurveySentiment/index.tsx [hunk 1]
```diff
@@ -16,8 +16,8 @@ interface Props {
 
 const SurveySentiment: React.FC<Props> = ({feel, activeFeel, onPress}) => {
     const handlePress = useCallback(() => {
-        onPress(feel);
-    }, [feel, onPress]);
+        feel === activeFeel ? onPress('') : onPress(feel);
```
Comment (timilsinabishal) [inline src/components/SurveySentiment/index.tsx]: null instead of ''?

File: src/screens/SurveyItem/Comment/index.tsx [hunk 2]
```diff
@@ -74,7 +73,6 @@ const Comment: React.FC<Props> = ({surveyId, commentItem, refetch}) => {
         DislikeCommentMutationVariables
     >(DISLIKE_COMMENT, {
         onCompleted: () => {
-            Toast.show('Comment disliked Successfully !');
```
Comment (timilsinabishal) [inline src/screens/SurveyItem/Comment/index.tsx:77]: What will happen to replied comments if the parent one is deleted?

File: src/components/SurveyReview/index.tsx [hunk 3]
```diff
@@ -62,8 +62,8 @@ export const SurveyReviewIcon = ({name, isActive}: ReviewIconProps) => {
 
 const SurveyReview: React.FC<Props> = ({name, activeReview, onPress}) => {
     const handlePress = useCallback(() => {
-        onPress && onPress(name);
-    }, [name, onPress]);
+        onPress && (name === activeReview ? onPress('') : onPress(name));
```
Comment (timilsinabishal) [inline src/components/SurveyReview/index.tsx]: null instead of ''?

File: src/screens/SurveyItem/Comment/index.tsx (hunk 2 above)
Comment (kamal09rumba) [reply src/screens/SurveyItem/Comment/index.tsx:77]: currently, the delete feature is not implemented yet. On the server side, if the parent comment object is deleted, also delete the objects that have references to it

Comment (timilsinabishal) [reply src/screens/SurveyItem/Comment/index.tsx:77]: Maybe we can add an is_deleted attribute to the and just flag those deleted comments. On the mobile side, need to use proper design so user understands that the comment is deleted.

**Review by timilsinabishal — APPROVED** [review]

**Pull request merged**